src/api/wallet.js

```diff
@@ -34,6 +34,35 @@ const walletApi = {
     });
   },
 
+  /**
```

Comment:
We may have to reduce the time limit in `getAddressHistory` in this file. MR #52 may have changed the time limit to 40 seconds and we may need to reduce it to the default value.

Reply:
Done